src/OpenSSL/crypto.py

```diff
@@ -1870,17 +1916,16 @@ def all_reasons(self):
         strings.
 
         :return: A list of reason strings.
-        :rtype: :py:class:`list` of :py:class:`bytes`
+        :rtype: :class:`list` of :class:`bytes`
         """
         return self._crl_reasons[:]
 
     def set_rev_date(self, when):
         """
         Set the revocation timestamp.
 
-        :param when: The timestamp of the revocation, as ASN.1 GENERALIZEDTIME.
-        :type when: :py:class:`bytes`
-        :return: :py:const:`None`
+        :param bytes when: The timestamp of the revocation, as ASN.1 GENERALIZEDTIME.
```

Comment:
flake8 doesn’t like this: https://travis-ci.org/pyca/pyopenssl/jobs/135324100#L233